### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by brlin-tw**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -138,7 +138,9 @@ parts:
       export ALSA_CONFIG_PATH="\$SNAP/etc/alsa.conf"
       exec "\$@"
       EOF
-    install: |
+
+      snapcraftctl build
```
Comment (lucyllewy) [inline snapcraft.yaml]: with the `nil` plugin this line is not needed

File: snapcraft.yaml [hunk 2]
```diff
@@ -106,7 +106,7 @@ parts:
   alsa:
     plugin: nil
     # after: [alsa-plugins, alsa-lib]
-    prepare: |
+    override-build: |
```
Comment (lucyllewy) [inline snapcraft.yaml]: Please move the `cat > alsa.conf` block to a new `override-pull` instead.

**Review by lucyllewy — CHANGES_REQUESTED** [review]

File: snapcraft.yaml (hunk 2 above)
Comment (brlin-tw) [reply snapcraft.yaml]: Shouldn't all heredocs blocks belong to `override-pull`?

**Comment by brlin-tw** [comment]
I've made the requested changes aside from moving all heredoc blocks to `override-pull`, please review.

**Review by lucyllewy — APPROVED** [review]

**Comment by lucyllewy** [comment]
Thank you :-)

**Pull request merged**